osi_environment.proto

```diff
@@ -110,7 +110,22 @@ message EnvironmentalConditions
     }
 
     // Definition of ambient illumination states.
+    // Ambient light is any light in the vehicle's environment not emitted by the vehicle 
+    // itself. It can include sun/moon light, light from other cars, street lights, etc.
+    // 
+    // Lux is the SI unit of luminance or illumination of a one metre square area, which is 
+    // equal to one lumen per square meter[5]. 1lx = 1lm/m²
+    // The lumen (symbol: lm) is the SI derived unit of luminous flux, a measure of the total
+    // quantity of visible light emitted by a source. The lumen is defined in relation to the candela as 1lm=1cd.sr
     //
+    //Categorization is done based on natural day/night time illuminance levels [6] and standards for required lighting levels on roads [6, 7, 8, 9]. 
+    //
+    // \par References:
+    // [5] [The NIST Reference on Constants, Units, and Uncertainty](https://physics.nist.gov/cuu/Units/units.html)
+    // [6] [National Optical Astronomy Observatory](https://www.noao.edu/education/QLTkit/ACTIVITY_Documents/Safety/LightLevels_outdoor+indoor.pdf)
+    // [7] [Standards for required street lighting in the USA](http://www.wsdot.wa.gov/research/reports/fullreports/847.1.pdf)
+    // [8] [Canadian IES-RP-8 standards for road lighting - municipality of Saint-Gédéon-de-. Beauce](http://sslnet.ca/wp-content/uploads/2011/10/LTE-RT-2011-0076-Anglais.pdf)
```

Comment:
Have you checked the doxygen output here?
It seems ok, but I would recommend to compile it once